src/NetCDFIO.jl

```diff
@@ -32,8 +36,8 @@ function NetCDFIO_Stats(nc_filename, output_interval, z_faces, z_centers)
         NC.defVar(profile_grp, "zc", z_centers, ("zc",))
         NC.defVar(profile_grp, "t", Float64, ("t",))
 
-        NC.defVar(profile_grp, "density", FT, ("zc","t"))      # TODO - not here
-        NC.defVar(profile_grp, "temperature", FT, ("zc","t"))  # TODO - not here
+        NC.defVar(profile_grp, "density", FT, ("zc","t"))
```

Comment:
Future To-Do: take as argument a dictionary of desired output variables.

Reply:
opened and issue https://github.com/CliMA/Kinematic1D.jl/issues/22